openwrt-feed/luci-app-usrmanage/htdocs/luci-static/resources/view/system/usrmanage.css

```
@@ -0,0 +1,50 @@
+	color: var(--error-color-high);
+}
+
+.usrmanage-pwcheck-mark {
+	flex: 0 0 1.15em;
+	width: 1.15em;
+	height: 1.15em;
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	border-radius: 999px;
+	border: 1.5px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)